playbooks/openstack/post-install.md

```diff
@@ -182,40 +189,38 @@ Note that in order for this example to work in your deployment, your servers mus
 
 ### Add Extra Docker Registry URLs
 
-This playbook is located in the [custom-actions](https://github.com/openshift/openshift-ansible-contrib/tree/master/playbooks/provisioning/openstack/custom-actions) directory.
+This playbook is located in the [sample-post-install-actions](https://github.com/openshift/openshift-ansible/tree/master/playbooks/openstack/sample-post-install-actions) directory.
```

Comment:
I think the original case for custom-actions was not limited to post install only cases. It can be pre-provision as well, and more cases perhaps.